pkg/tools/log/logruslogger/formatter/formatter.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2021 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package formatter contains custom logrus formatter that duplicates fields and other metadata from nested-logrus-formatter to each line of the message
+package formatter
+
+import (
+	"bytes"
+	"strings"
+
+	nested "github.com/antonfisher/nested-logrus-formatter"
+	"github.com/sirupsen/logrus"
+)
+
+// Formatter - implements logrus.Formatter
+// Duplicates fields and other metadata from nested-logrus-Formatter to each line of the message
+type Formatter struct {
+	nf nested.Formatter
+}
+
+// New creates new Formatter and initializes it
+func New() *Formatter {
+	f := Formatter{}
+	f.nf.FieldsOrder = []string{"id", "name"}
+	return &f
+}
```

Comment:
Why is this public?

Reply:
it's separate package - we need to use method New() in logruslogger package
should i change this?

Reply:
I think yes, we should hide implementation details if no one uses them externally.

Reply:
ok, done